test/tubes/serialtube_test.rb

```diff
@@ -0,0 +1,131 @@
+# encoding: ASCII-8BIT
+
+require 'open3'
+
+require 'test_helper'
+
+require 'pwnlib/tubes/serialtube'
+
+class SerialTest < MiniTest::Test
+  include ::Pwnlib::Tubes
+
```

Comment:
Is the serial tube intend to work on macOS?
If yes, let the tests pass on macOS (install socat)
If no, skip the test unless the environment is on Linux

Reply:
And skip on Windows as well

Reply:
To install socat, simply add `brew install socat` in travis/install.sh#setup_osx

Is that possible to test serialtube on Windows? I said skip on Windows because the tests you added (`socat ...`) can't be supported on Windows.

If serialtube is supposed to support on Windows, maybe you should try another testing method to make it work on Windows as well.
Otherwise, just forget Windows supports and skip the tests.